readme-oracle-tests.md

```diff
@@ -3,20 +3,18 @@ https://download.liquibase.org/download/
 Set environment variable LIQUIBASE_HOME
 Copy driver to $LIQUIBASE_HOME/lib
 
-#### Run docker with Oracle XE 11 release 2
+#### Run docker with Oracle XE 18c (you need to build it first)
 ```
 cd sql/oracle
-rm -f init/.cache
-docker run --rm -it -p 1521:1521 -v $PWD/init:/docker-entrypoint-initdb.d --name morejdbc-oracle wnameless/oracle-xe-11g-r2
+docker run --name morejdbc-oracle -d -p 1521:1521 oracle/database:18.4.0-xe
```

Comment:
this image is not found in public repo without login

Reply:
It's true. You need to build it as I wrote at the heading: **Run docker with Oracle XE 18c (you need to build it first).**
There is no way to use prepared images _officially_. The only way to use image - store it in private repo as I did it for CI.